test/sql/insert/test_big_insert.test

```diff
@@ -6,6 +6,9 @@
 statement ok
 CREATE TABLE integers(i INTEGER)
 
+#FIXME : any reason we don't just use this?
```

Comment:
Yes, this is intentionally testing many expressions (>1050) - although it should probably be updated to 2050 now that we have increased the vector size.